packages/driver/src/cy/snapshots.coffee

```diff
@@ -47,7 +47,11 @@ getStylesFor = (doc, $$, stylesheets, location) ->
       }
     else
       ## otherwise, it's a style tag, and we can just grab its content
-      makePathsAbsoluteToDoc(doc, $$(stylesheet).text())
+      styleRules = $$(stylesheet).text() ||
```

Comment:
Is it possible a stylesheet could have text inside it and inserted rules? If so, we should account for that case too.

Reply:
If <style> has text inside then text is converted into style rules:

![image](https://user-images.githubusercontent.com/3865478/40852734-bca318c6-65cb-11e8-93d4-79a750d03c29.png)

I didn't want to change it only for `sheet.rules.map()` because already existing tests were not pass (as you can see browser optimises css rule, add `;` and change colour values.

What do you think ?

Reply:
@chrisbreiding Check the latest commit and let me know what do you think.

Reply:
Looks good to me. Thanks!